docgen/src/Styling.md

```diff
@@ -6,62 +6,174 @@ nav_groups:
 nav_sort: 1
 ---
 
+Welcome to the react-instantsearch theming tutorial!
```

Comment:
I am sorry to say it only now, but should all our guide pages be tutorials? Can't they just be simple guide pages with different use cases and explanations?

Tutorial requires a lot more focus and will to follow than a simpler documentation on styling.

If all I want is "Learn more about styling" then it's unlikely I want to follow a complete tutorial?